src/_actions/AccountActions.js

```diff
@@ -14,3 +14,17 @@ export const serverDataPayoutCurrencies = serverResponse => ({
     type: types.SERVER_DATA_PAYOUT_CURRENCIES,
     serverResponse,
 });
+
+export const serverAuthFailed = serverResponse => ({
```

Comment:
Do we need this? We already handle server response for auth, and it can be a success or failure?